src/main/scala/com.snowplowanalytics.sauna/apis/UrbanAirship.scala

```diff
@@ -0,0 +1,166 @@
+package com.snowplowanalytics.sauna
+package apis
+
+
+
+import java.io.DataOutputStream
```

Comment:
Lots of unnecessary imports.
When you're putting imports in your module it's a signal on what libs (approaches) you're using. And clearly unused imports is wrong signal. All modern IDEs (such as Inellij IDEA and Eclipse) notify you about unused imports.
Also, when you clean it up, it would be a good idea to group imports by their library as it was done in previous [modules](https://github.com/snowplow/sauna/blob/master/src/main/scala/com.snowplowanalytics.sauna/responders/sendgrid/Recipients.scala#L17-L38)